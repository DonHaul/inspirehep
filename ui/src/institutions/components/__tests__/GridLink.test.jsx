import React from 'react';
import { render } from '@testing-library/react';
import GridLink from '../GridLink';

describe('GridLink', () => {
  it('renders', () => {
    const grid = 'grid123';
    const { asFragment } = render(<GridLink grid={grid} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
