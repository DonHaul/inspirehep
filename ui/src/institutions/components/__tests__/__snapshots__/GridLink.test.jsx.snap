// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridLink renders 1`] = `
<DocumentFragment>
  <span>
    GRID Record: 
    <a
      href="//grid.ac/institutes/grid123"
      target="_blank"
    >
      grid123
    </a>
  </span>
</DocumentFragment>
`;
